.travis.yml

```diff
@@ -1,26 +1,45 @@
-sudo: enabled
-
+dist: xenial
+sudo: required
 language: c
 
 install:
-    - make build-depends
-    - sudo apt-get -y install debian-archive-keyring dpkg=1.17.5ubuntu5.8
+  - make build-depends
+  - sudo apt-get -y install debian-archive-keyring
 
 # See the Ubuntu section in the debian/README file for more details, but this
 # is a good example of why CI is useful /before/ shipping a product..
 #
 # Also, the age of the debian-archive-keyring (2012) is simply concerning. So,
 # I have to manually add the debian stable release keys
 before_script:
-    - sudo sed -i '989s/$forceyes//' /usr/sbin/multistrap
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
-    - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 7638D0442B90D010
+  - sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EF0F382A1A7B6500
 
 script:
-    - make test CONFIG_DISABLE_KVM=yes
-    - sha1sum output/*
-    - ls -al output/*
+  - make test CONFIG_DISABLE_KVM=yes
+  - sha1sum output/*
+  - ls -al output/*
+
+after_success:
+  - rename "s/.img/-$TRAVIS_TAG.img/" output/*.img
 
-# TODO
-# - auto deploy the output files to a github release if this is a tagged commit
-# - Push the rendered document svg + pdf output to somewhere
+deploy:
+  - provider: releases
+    api_key:
+      secure: FIKQFkf9xcBvom8Chf/NMtrb1hTNHScZSORBWKbq0gBU0fQZDvOvxXFPWuyq98g0uFaj7Tk9t6+g/fxrkZ76ZINSDZ6aIZBPLWU8iChMiG4kOgCvwN5zQyK4Xbr0wlNNaAdz1UQySIbKhpbd8MV5tHbU1HHskDDAi0nznKO9kVk+Fw/lTYzjAr/NrNZfyAgdaflVLgFr4UEoIx3pxlcm3RJbzApdrCSdgz1WzIQxtCCDHXWbAwDMxvyh42al1E11rbFSXJgQp1fjo4h3I6FplOd+4BRotYNcLEwJYEQBKbwyGgIcvjMlXosGjQhPxeSN5/525XRwn8C9mB5uFFMbkO5X24cEc9JYqpQg+ArbxSvDmNEu14up3jztSb72Eo7HlPZP3JZPp6uWk6fyZReBd20QvQYOTEb7JHr/yfJsUCYPIp/AsAKW0D85HHjVXcKZ1q3J4pRHMDJZS4QB9MmYyAknRD+EWDuD9Hu+v+TgX/F1TSPov4yRojI1LI5T16vqwAuBNkLzQpTy5OWx4ikOgqYZqjV1oVgITtkrQP5wUjezLe8mVrsDl8SyGfPvbnbnbSqplEcjjrbREVDeFesRMtwauRBQNZR16fS7OG6GWp1GporiFqrW10wpVM11xRqbiZurKbyi1mhYqGTjQhXPrToJUP+8fSw6zXoFAdDQPWQ=
+    file_glob: true
+    file: "output/*"
+    skip_cleanup: true
+    on:
+      repo: tomeshnet/mesh-orange
+      tags: true
+  - provider: pages
+      github-token:
+        secure: FIKQFkf9xcBvom8Chf/NMtrb1hTNHScZSORBWKbq0gBU0fQZDvOvxXFPWuyq98g0uFaj7Tk9t6+g/fxrkZ76ZINSDZ6aIZBPLWU8iChMiG4kOgCvwN5zQyK4Xbr0wlNNaAdz1UQySIbKhpbd8MV5tHbU1HHskDDAi0nznKO9kVk+Fw/lTYzjAr/NrNZfyAgdaflVLgFr4UEoIx3pxlcm3RJbzApdrCSdgz1WzIQxtCCDHXWbAwDMxvyh42al1E11rbFSXJgQp1fjo4h3I6FplOd+4BRotYNcLEwJYEQBKbwyGgIcvjMlXosGjQhPxeSN5/525XRwn8C9mB5uFFMbkO5X24cEc9JYqpQg+ArbxSvDmNEu14up3jztSb72Eo7HlPZP3JZPp6uWk6fyZReBd20QvQYOTEb7JHr/yfJsUCYPIp/AsAKW0D85HHjVXcKZ1q3J4pRHMDJZS4QB9MmYyAknRD+EWDuD9Hu+v+TgX/F1TSPov4yRojI1LI5T16vqwAuBNkLzQpTy5OWx4ikOgqYZqjV1oVgITtkrQP5wUjezLe8mVrsDl8SyGfPvbnbnbSqplEcjjrbREVDeFesRMtwauRBQNZR16fS7OG6GWp1GporiFqrW10wpVM11xRqbiZurKbyi1mhYqGTjQhXPrToJUP+8fSw6zXoFAdDQPWQ=
+    local-dir: docs
```

Comment:
Maybe, instead of simply deploying all the crap in the docs dir, there should be a make target that creates the required outputs and then those documents can be deployed (ie, we done need to deploy the Makefile and probably dont get value from deploying the .dot source files)

Reply:
Deploying the `.dot` is probably okay. Yes we currently do not deploy the `Makefile`.

I agree a new make target deploying to `output-docs`, without changing `docs`, will be most consistent with how we handle actual images.

Reply:
Now everything is generated into `output-docs` directly and `docs` stays unmodified.